src/commands/ext-install.ts

```diff
@@ -83,23 +83,22 @@ async function installExtension(options: InstallExtensionOptions): Promise<void>
       `successfully installed ${clc.bold(spec.displayName || spec.name)}, ` +
         `its Instance ID is ${clc.bold(instanceId)}.`
     );
-    const usageInstruction =
-      _.get(response, "config.populatedPostinstallContent") ||
-      populatePostinstall(source.spec.postinstallContent || "", params);
-    if (usageInstruction) {
-      utils.logLabeledBullet(logPrefix, `usage instructions:\n${marked(usageInstruction)}`);
-    } else {
-      logger.debug("No usage instructions provided.");
-    }
     utils.logLabeledBullet(
       logPrefix,
       marked(
-        `You can view your new instance in the Firebase console: ${utils.consoleUrl(
+        `View your new instance in the Firebase console: ${utils.consoleUrl(
           projectId,
           `/extensions/instances/${instanceId}?tab=usage`
         )}`
       )
     );
+    logger.info(
+      marked(
+        "Learn how to manage your installed extension, including how to monitor, reconfigure, and update to new versions: " +
+          "https://firebase.google.com/docs/extensions/manage-installed-extensions?platform=cli \n"
```

Comment:
This provides similar info as running "firebase ext". I would only include one of these lines, so we don't overwhelm users. 

If I had to pick one of the 2, I would pick telling users about "firebase ext". I'm hesitant to link to specific docs pages for a new product since those links may change in the future, and once a CLI version is out, it is out forever, so it may have a broken link. On the other hand, since "firebase ext" is another CLI command, we can guarantee it will work since it is downloaded code.

Reply:
I can get behind this - however, I took this copy exactly from what @rachelsaunders posted in the internal bug. Rachel, what do you think about this?

Reply:
Yep, I'd +1 Lauren's suggestion.

Reply:
I'd probably combine the two sections and say:
You can run `firebase ext` to view available Firebase Extensions commands including those to manage your installed extension (like reconfiguring and updating to new versions).